Resources/Prototypes/Entities/Structures/Machines/chem_master.yml

```
@@ -17,6 +17,13 @@
   - type: Icon
     sprite: Structures/Machines/mixer.rsi
     state: mixer_loaded
```

```suggestion
    state: mixer_empty
```
Should use the empty sprite for UI previews then since mixer_loaded is only the beaker now.